api.js

```diff
@@ -1,10 +1,16 @@
 import axios from "axios";
 
-export function getArticles() {
+export function getArticles(topic) {
+    let fetchUrl = `https://h-nc-news-project.onrender.com/api/articles/`
+    if (topic) {
+        fetchUrl += `?topic=${topic === "all" ? "" : topic}`
```

Comment:
Ah - this part isn't needed! Axios params will really nicely ignore any values that aren't present in the request.